Stylesheets/Components/badge.css

```diff
@@ -0,0 +1,81 @@
+.badge{
+    position: relative;
+    margin:1rem;
+ 
+}
+
+.badge i{
+    font-size: 2rem;
+    position: relative;
+    display: inline-block;
+ 
+}
+.badge span{
+    display: inline-block;
+    background-color: var(--st-span-text);
+    color: var(--st-white);
+    font-weight: bold;
+    text-align: center;
+    width:1.3rem;
+    height:1.3rem;
+    border-radius: 50%;
+    position:absolute;
+    top:0;
+    right:0;
+}
+.badge img{
+    position: relative;
+    display: inline-block;
+ 
+}
+.online{
```

Comment:
Follow specific naming conventions. This can be given as `badge-online`.
(Apply in all)

Reply:
Thanks a lot for pointing it out, it makes more sense and i will make changes